docs/api.md

```diff
@@ -465,6 +468,66 @@ A valid request will return JSON with these properties:
 }
 ```
 
+### GET /v1/user/tokens
+
+Gets a list of active tokens for a user. The `token_id` field of each token
+record is an opaque identifier that can be used to revoke the token.
+
+**Required scope:** `oauth:tokens`
+
+#### Request Parameters
+
+**Example:**
+
+```sh
+curl -v \
+-H "Authorization: Bearer 558f9980ad5a9c279beb52123653967342f702e84d3ab34c7f80427a6a37e2c0" \
+"https://oauth.accounts.firefox.com/v1/user/tokens"
+```
+
+#### Response
+
+A valid response will have a 200 status code and a list of token records:
+
+```json
+{
+  "tokens": [{
+    "token_id": "92784bc8b18985ae03a015bff9e6b6322e9a74a9673fa08bada418e4fb4e2f6a",
```

Comment:
I suggest "id" instead of "token_id" for symmetry with the existing `/clients` endpoint.